src/main/webapp/app/core/legal/data-export/data-export.component.ts

```diff
@@ -96,10 +95,7 @@ export class DataExportComponent implements OnInit {
     }
 
     downloadDataExport() {
-        this.dataExportService.downloadDataExport(this.dataExportId).subscribe((response: HttpResponse<Blob>) => {
-            downloadZipFileFromResponse(response);
-            this.alertService.success('artemisApp.dataExport.downloadSuccess');
```

Comment:
Is this message just gone? In that case, the translation should also be removed

Reply:
yes, it is, removed it.